packages/fx-core/src/component/resource/spfx/depsChecker/generatorChecker.ts

```diff
@@ -70,6 +73,40 @@ export class GeneratorChecker implements DependencyChecker {
     return ok(true);
   }
 
+  public async ensureLatestDependency(
+    ctx: PluginContext | ContextV3
+  ): Promise<Result<boolean, FxError>> {
+    telemetryHelper.sendSuccessEvent(ctx, TelemetryEvents.EnsureLatestSharepointGeneratorStart);
+    let needInstall = false;
+    try {
+      needInstall = !(await this.isLatestInstalled());
```

Comment:
In our new design, why not just execute 'npm install <package>@latest' all the time since we don't care about what latest version is?

Reply:
I think it will help save some time :)